### ISSUE TITLE
Hunter pet gone after player is trapped in cthun stomach after raid wipe

### ISSUE THREAD

**Issue by couchsitter** [elysium-project/server#2438]
While I was outside the body of Cthun before being eaten in phase 2, my pet was active. When I was eaten by the tentacle my pet was dismissed and not active when I entered the stomach. I summoned my pet to assist in dealing damage to the inner two tentacles and outside of the body the raid wiped. After being taken out of combat and the tentacles disappearing and the poison debuffs dropped, I /camped out. Upon logging in again I was at the graveyard outside the instance even though I did not die which is a minor setback but the raid DID wipe so it makes sense. When I zoned back in to AQ40 and called for my pet, nothing happened. After a conversation with GM Pottu he informed me that my pet was teleported to the stable. When raid was over I went to a Stable master and cannot see my pet.

I have multiple pets with the same name and the pet I was using was not there. Screenshot is attached.
![image](https://user-images.githubusercontent.com/8251204/30146369-a1e977d4-9365-11e7-9b9d-db07b06703dc.png)
Realmplayers Pets - Bellygrub and Wolfbud have been released, I should have a level 60 Wind Serpent and a level 60 Cat both named Galaurien.

![lonely_pet](https://user-images.githubusercontent.com/8251204/30146344-6e852dca-9365-11e7-9bd2-69b136b11c17.png)
I should have a Son of Hakkar with loyalty level 6 as my active raiding pet and a level 60 Cat with loyalty level 6 as my PvP pet. Looking in the stables I only see the cat. 

It's not the end of the world as I can always go train another one but it is an issue that other hunters might encounter and with a rarer pet / specific pet, it might become more of a hassle.

**Comment by couchsitter** [comment]
TL;DR My pet I was using is gone, I can't summon/dismiss/rez it via macro or clicking the spell out of the spellbook.

**Comment by Mostprovidedlinks** [comment]
Can someone test this pls.so if pet is fixed and it is working

**Comment by Spionida** [comment]
Read the bit that says "Occasionally a game bug will lose them (pets). I gave you two possibilities. Read them both. The GMs can not, not will not, retrieve pets.
